modules/container-registry/registries/main.bicep

```diff
@@ -298,6 +301,15 @@ resource registry 'Microsoft.ContainerRegistry/registries@2022-02-01-preview' =
   }
 }
 
+resource cacheRules 'Microsoft.ContainerRegistry/registries/cacheRules@2023-01-01-preview' = [for repo in repoCaches: {
```

Comment:
Hi @davidkarlsen, thanks a lot for this contribution!

cacheRules is a registry child resource and as such should be deployed as a child module. To align, the process would be as follows:
- create a registry subfolder called `cache-rules`
- the subfolder should contain:
   - a `main.bicep` template deploying the cacherule resource. you can reference other registry child resources such as replications or webhooks
   - a `version.json` that you can copy from any other module and put at version 1.0
   - a `metadata.json` containing module title and description. You can reference any other module metadata.json file 
   - a `README` file that you can generate through the [Set-ModuleReadMe utility ](https://github.com/Azure/ResourceModules/blob/main/utilities/tools/Set-ModuleReadMe.ps1)
 - reference the new child module cache-rules from the parent registry, similarly to how done for registry_replications and registry_webhooks
 - update the parent registries readme file using the same utility as above

This sounds like a lot of work, we'd be very grateful if you'd have the availability to contribute to it

Reply:
fixed